content/donate.adoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: "Donate to Jenkins"
+---
+
+NOTE: You can donate directly through the
```

Comment:
Which isn't a good practice for the aforementioned reason. Probably better to use one line per sentence (if Asciidoc supports that, that's what I generally do with Markdown) and use soft wraps in your editor.